### ISSUE TITLE
Double scrobble

### ISSUE THREAD

**Issue by jeromefaria** [ethanndickson/iina-lastfm#10]
Hi,

For some reason, my tracks are getting scrobbled twice. At first I was running: 
```bash
iina --mpv-shuffle $path
```
I thought maybe because of using the mpv flag, that could be the reason. But I also tried:
```bash
mpv --shuffle $path
```
And got the same result, which is to get two scrobbles seconds apart. See picture:
![image](https://github.com/ethanndickson/iina-lastfm/assets/2834077/ff7fa2a8-c07e-4301-8c2a-0b8e75b41c0a)

Also, on a side note, I had to manually run:
```bash
perl -MCPAN -e 'install Bundle::LWP'
```
To install a missing module, otherwise the script wouldn't run out of the box.

My system information is:
```bash
ProductName:	macOS
ProductVersion:	12.6.7
BuildVersion:	21G651
```
I hope this helps. Thanks!

**Comment by ethanndickson** [comment]
> For some reason, my tracks are getting scrobbled twice.

I hadn't seen this happen before, so I assumed it was a Monterey issue. I literally just updated, and can now reproduce the issue. Totally perplexing - I'll look into it.


> To install a missing module, otherwise the script wouldn't run out of the box.

Likewise, super strange. Although LWP::UserAgent isn't core perl, my understanding is that it came with macOS. It's possible that this changed in Monterey. 
Unfortunately, I don't have a clean install to check this with. I will either update the documentation, or see if it can be rewritten with just the core perl HTTP library.

**Comment by ethanndickson** [comment]
Oops, I had created a duplicate lua script in the scripts folder and then forgot to delete it after updating. With just a single script file on Monterey I am unable to reproduce this.

**Comment by ethanndickson** [comment]
> Hi @ethanndickson
> 
> Did you ever get around to fixing this? Or should I delete one of the scripts myself? If so, which one?
> 
> Thanks!

Apologies, when I said I had created a duplicate script, I meant locally.
Regardless, thank you for the ping, as I was able to reproduce this, and it looks like an easy fix.

mpv exposes two similar events:
> `start-file`: Happens right before a new file is loaded. When you receive this, the player is loading the file (or possibly already done with it).

> `file-loaded`:  Happens after a file was loaded and begins playback.
    
Currently, the lua script uses `start-file`, which appears to be liable to being called multiple times, but only under specific circumstances, i.e. shuffle? I assume one to load the entire playlist, and another for the file itself? 

Switching to `file-loaded` only sends a single event when shuffling, and doesn't appear to break anything else, but I can't confirm this without trawling through the mpv source code. I might try that later.

For now, I'll commit a change from `start-file` to `file-loaded` which should fix it on your device, let me know if you have any issues :)
